apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -58,6 +60,16 @@ const Ticket = new GQLListSchema('Ticket', {
                 create: false,
             },
         },
+        reviewValue: {
+            schemaDoc: 'Review of the ticket by a resident on a 2-point scale. 0 – ticket returned, 1 – bad review, 2 – good review',
+            type: Select,
+            dataType: 'string',
+            options: '0, 1, 2',
```

Comment:
is it a string? probably it's better to use some keyword. good, bad, ..

Reply:
or use the numbers

Reply:
used enums 'returned, good, bad'